experimenter/experimenter/nimbus-ui/src/components/Summary/TableAudience/index.tsx

```diff
@@ -230,4 +243,23 @@ const TableAudience = ({ experiment }: TableAudienceProps) => {
   );
 };
 
+interface ExperimentListProps {
+  experiments: getExperiment_experimentBySlug_excludedExperiments[];
+}
+function ExperimentList({ experiments }: ExperimentListProps) {
+  if (experiments.length === 0) {
+    return <span>None</span>;
+  }
+
+  return (
+    <ul className="m-0">
+      {experiments.map((experiment) => (
+        <li key={experiment.slug}>
+          <a href={`/nimbus/${experiment.slug}/summary`}>{experiment.slug}</a>
+        </li>
```

Comment:
Oh good yes this is helpful, but the spacing and dot with the `<li>`s takes so much space, can we put these in `<p>`s instead?  And use the name instead of the slug? 🙏

Reply:
Done!